src/docs/asciidoc/web/webmvc.adoc

```diff
@@ -1634,15 +1634,15 @@ extracts the name, version, and file extension:
 .Java
 ----
 	@GetMapping("/{name:[a-z-]+}-{version:\\d\\.\\d\\.\\d}{ext:\\.[a-z]+}")
-	public void handle(@PathVariable String version, @PathVariable String ext) {
+	public void handle(@PathVariable name, @PathVariable String version, @PathVariable String ext) {
```

Comment:
FYI: you accidentally deleted the `String` type declaration for the `name` parameter.

But don't worry about it: I've fixed it locally prior to merging.

```suggestion
	public void handle(@PathVariable String name, @PathVariable String version, @PathVariable String ext) {
```

Reply:
Sorry for that and thanks a lot. I will be careful next time.